orchestrator/orchestrator.go

```diff
@@ -143,7 +154,8 @@ func (o *Orchestrator) NetworkUp(ctx context.Context, network string) (NetworkDe
 	n.PeerKey = s.PeerKey
 	n.SwarmKey = string(opts.SwarmKey)
 	n.SwarmAddr = fmt.Sprintf("%s:%s", o.address, newNode.Ports.Swarm)
-	n.Activated = time.Now()
+	var now = time.Now()
+	n.Activated = &now
```

Comment:
couldn't this be `n.Activated = &time.Now()`?

Reply:
its not allowed haha

![image](https://user-images.githubusercontent.com/23356519/53691437-e750c480-3d32-11e9-83c9-5fe81e23515d.png)